prime-router/metadata/tables/local/fhirpath_filter_shorthand.csv

```diff
@@ -2,11 +2,14 @@ variable,fhirPath
 processingId,Bundle.entry.resource.ofType(MessageHeader).meta.tag.where(system = 'http://terminology.hl7.org/CodeSystem/v2-0103').code
 messageId,Bundle.entry.resource.ofType(MessageHeader).id
 patient,Bundle.entry.resource.ofType(Patient)
+performer,Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve()
 performerState,Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state
 patientState,Bundle.entry.resource.ofType(Patient).address.state
 specimen,Bundle.entry.resource.ofType(Specimen)
 serviceRequest,Bundle.entry.resource.ofType(ServiceRequest)
 observation,Bundle.entry.resource.ofType(Observation)
 provenance,Bundle.entry.resource.ofType(Provenance)
 hl7MessageType,Bundle.entry.resource.ofType(Provenance).activity.coding.code
-obsPerformedCodes,%resource.code.coding.code
\ No newline at end of file
+obsPerformedCodes,%resource.code.coding.code
+LACountyCities,'Acton' | 'Agoura Hills' | 'Alhambra' | 'Altadena' | 'Arcadia' | 'Arleta' | 'Artesia' | 'Avalon' | 'Azusa' | 'Baldwin Park' | 'Bell' | 'Bell Gardens' | 'Bellflower' | 'Beverly Hills' | 'Burbank' | 'Calabasas' | 'Canoga Park' | 'Canyon Country' | 'Carson' | 'Castaic' | 'Cerritos' | 'Chatsworth' | 'City Of Industry' | 'Claremont' | 'Compton' | 'Covina' | 'Culver City' | 'Diamond Bar' | 'Downey' | 'Duarte' | 'El Monte' | 'El Segundo' | 'Encino' | 'Gardena' | 'Glendale' | 'Glendora' | 'Granada Hills' | 'Hacienda Heights' | 'Harbor City' | 'Hawaiian Gardens' | 'Hawthorne' | 'Hermosa Beach' | 'Huntington Park' | 'Inglewood' | 'La Canada Flintridge' | 'La Crescenta' | 'La Mirada' | 'La Puente' | 'La Verne' | 'Lake Hughes' | 'Lakewood' | 'Lancaster' | 'Lawndale' | 'Littlerock' | 'Llano' | 'Lomita' | 'Long Beach' | 'Los Angeles' | 'Lynwood' | 'Malibu' | 'Manhattan Beach' | 'Marina Del Rey' | 'Maywood' | 'Mission Hills' | 'Monrovia' | 'Montebello' | 'Monterey Park' | 'Montrose' | 'Mount Wilson' | 'Mt Baldy' | 'Newhall' | 'North Hills' | 'North Hollywood' | 'Northridge' | 'Norwalk' | 'Pacific Palisades' | 'Pacoima' | 'Palmdale' | 'Palos Verdes Peninsula' | 'Panorama City' | 'Paramount' | 'Pasadena' | 'Pearblossom' | 'Pico Rivera' | 'Playa Del Rey' | 'Pomona' | 'Rancho Palos Verdes' | 'Redondo Beach' | 'Reseda' | 'Rosemead' | 'Rowland Heights' | 'San Dimas' | 'San Fernando' | 'San Gabriel' | 'San Marino' | 'San Pedro' | 'Santa Clarita' | 'Santa Fe Springs' | 'Santa Monica' | 'Sherman Oaks' | 'Sierra Madre' | 'Signal Hill' | 'South El Monte' | 'South Gate' | 'South Pasadena' | 'Stevenson Ranch' | 'Studio City' | 'Sun Valley' | 'Sunland' | 'Sylmar' | 'Tarzana' | 'Temple City' | 'Toluca Lake' | 'Topanga' | 'Torrance' | 'Tujunga' | 'Universal City' | 'Valencia' | 'Valley Village' | 'Valyermo' | 'Van Nuys' | 'Venice' | 'Verdugo City' | 'Walnut' | 'West Covina' | 'West Hills' | 'West Hollywood' | 'Westlake Village' | 'Whittier' | 'Wilmington' | 'Winnetka' | 'Woodland Hills'
```

Comment:
Not a big fan of this, have we thought of alternative implementations? Do we have a ticket? I fear this could make this file unwieldy in the future if we continue including sender or receiver specific shorthands in here?

Reply:
That was brought up during the design. We knew it was going to be hard to maintain this list as part of the shorthands. I think the alternatives are listed under the design, but this was the one that would that required the less work. We can revisit the design and create a ticket on one of the alternatives.